specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -51,6 +51,555 @@
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterWorkerSelector"
+          }
+        },
+        "scheduledTimeUtc": {
+          "description": "Router Job Received Scheduled Time in UTC",
+          "type": "string",
+          "format": "date-time"
+        },
+        "unavailableForMatching": {
+          "description": "Unavailable For Matching for Router Job Recieved",
+          "type": "boolean"
+        }
+      }
+    },
+    "AcsRouterJobSchedulingFailedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobSchedulingFailed event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventBaseProperties"
+        }
+      ],
+      "properties": {
+        "priority": {
+          "description": "Router Job Priority",
+          "type": "integer",
+          "format": "int32"
+        },
+        "attachedWorkerSelectorsExpired": {
+          "description": "Router Job Scheduling Failed Attached Worker Selector Expired",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterWorkerSelector"
+          }
+        },
+        "requestedWorkerSelectorsExpired": {
+          "description": "Router Job Scheduling Failed Requested Worker Selector Expired",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterWorkerSelector"
+          }
+        },
+        "scheduledTimeUtc": {
+          "description": "Router Job Scheduling Failed Scheduled Time in UTC",
+          "type": "string",
+          "format": "date-time"
+        },
+        "failureReason": {
+          "description": "Router Job Schecduling Failed Reason",
+          "type": "string"
+        }
+      }
+    },
+    "AcsRouterJobUnassignedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobUnassigned event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventBaseProperties"
+        }
+      ],
+      "properties": {
+        "assignmentId": {
+          "description": "Router Job Unassigned Assignment Id",
+          "type": "string"
+        },
+        "workerId": {
+          "description": "Router Job Unassigned Worker Id",
+          "type": "string"
+        }
+      }
+    },
+    "AcsRouterJobWaitingForActivationEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobWaitingForActivation event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventBaseProperties"
+        }
+      ],
+      "properties": {
+        "priority": {
+          "description": "Router Job Waiting For Activation Priority",
+          "type": "integer",
+          "format": "int32"
+        },
+        "attachedWorkerSelectorsExpired": {
```

Comment:
This name reads a little funny - would something like `expiredAttachedWorkerSelectors` and `expiredRequestedWorkerSelectors` (for the next property) make sense?